lib/jsdom/living/events/MouseEvent-impl.js

```diff
@@ -7,6 +7,19 @@ const UIEventImpl = require("./UIEvent-impl").implementation;
 const MouseEventInit = require("../generated/MouseEventInit");
 
 class MouseEventImpl extends UIEventImpl {
+  get x() {
+    return this.clientX;
+  }
+  get y() {
+    return this.clientY;
+  }
+  get offsetX() {
+    return 0;
+  }
+  get offsetY() {
+    return 0;
+  }
+
```

Comment:
You're missing getters for pageX and pageY. See how they are defined in steps 2-3 of https://drafts.csswg.org/cssom-view/#dom-mouseevent-pagex .

Reply:
i'm not sure if the getters are actually needed, since (i think) by default properties are readable/writable (e.g. see clientX usage above, which works and doesn't have an explicit getter)

Reply:
Done, and added a test